src/platform/plugins/shared/dashboard/server/api/create/create.ts

```diff
@@ -21,12 +21,13 @@ export async function create(
   createBody: DashboardCreateRequestBody
 ): Promise<DashboardCreateResponseBody> {
   const { core } = await requestCtx.resolve(['core']);
+  const { access_control: accessControl, ...restOfData } = createBody.data;
```

Comment:
This logic should also be added to the update endpoint so that update does not add access_control to saved object attributes.